apps/dashboard/app/models/batch_connect/session.rb

```diff
@@ -47,6 +47,10 @@ def get_binding
     # @return [String] script type
     attr_accessor :script_type
 
+    # Flag to know if a session has been manually canceled
+    # @return [Boolean] true if job is canceled
+    attr_accessor :canceled
```

Comment:
I can't tell if we need this attribute. Does it work the same without it?

Reply:
It was needed in the previous implementation. After some changes, this is no longer needed. I have removed